docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md

```diff
@@ -1,71 +1,82 @@
 
-This parameter is crucial for removing noise from chromatograms. If this parameter is set to, e.g.,
-50, the lowest 50 % of intensities will be removed. Since the _all_ retention time in the data file
-are used for this determination, this value should be rather high (e.g., 95 %) to begin with and
-only lowered if necessary.
+:octicons-alert-16: Since the algorithms examines the EICs throughout the entire RT range (_i.e._ also the zero data points are considered), we recommend to set this value rather high (_e.g._ 95%) and lower it only if necessary. 
+:octicons-alert-16: ION MOBILITY 80%
 
-##### Search minimum rt range (min)
+**Minimum search range RT/Mobility (absolute)**: size of the moving window examined for local minima search. Overly low values can cause peak edges to be cut off. On the other hand, too high might lead to incomplete separation of narrowly eluting compounds.
 
-Determines the step size that will be scanned for individual peaks. Setting this value too low, can
-cause peak edges to be cut off, setting it too high might lead to incomplete separation of narrowly
-eluting compounds.
 
-##### Minimum relative height
+**Minimum relative height**: minimum relative intensity (respect to the highest data point in the EIC) that a peak need to reach to be retained as a feature. Overly high thresholds may lead tolow-intensity features being discarded.
 
-Determines the minimum relative intensity of a individual feature in relation to the highest
-intensity in the chromatogram. May lead to discrimination of low intensity features.
+:octicons-light-bulb-16: **Tip**. Modern HRMS instruments can show linear dynamic ranges op to five orders of magnitude. If we take an Orbitrap device with a detector saturation level around 1.0E10 intensity, a _Minimum relative height_ = 0.001 would correspond to 1.0E5.
 
-##### Minimum absolute height
 
-Determines the minimum absolute intensity of a feature to be recognised by the algorithm. This
-parameter depends on what you want to detect, the instrument and detector type. Usually, Orbitrap
+**Minimum absolute height**: minimum absolute intensity that a peak needs to reach to be retained as a feature.
+
+This parameter depends on what you want to detect, the instrument and detector type. Usually, Orbitrap
 instruments report higher intensities that TOF instruments. However, the noise level is also higher
 for Orbitrap than for TOF instruments. For TOF instruments 1E3 or even 5E2 can be appropriate,
 whilst Orbitraps can require 1E5 or 5E4.
 
-##### Min ration of peak top/edge
+**Min ratio of peak top/edge**: Describes the minimum ratio of the highest point of a peak to the lowest point of a peak. This mostly affects detection of low intensity and not-baseline-resolved signals and should be optimised using such a signal as an example.
```

Comment:
We recommend values between 1.7 (not baseline separated) and 2 to start the optimisation.